src/keybindings.h

```diff
@@ -273,6 +273,7 @@ enum GeanyKeyBindingID
 	GEANY_KEYS_FORMAT_SENDTOCMD7,				/**< Keybinding. */
 	GEANY_KEYS_FORMAT_SENDTOCMD8,				/**< Keybinding. */
 	GEANY_KEYS_FORMAT_SENDTOCMD9,				/**< Keybinding. */
+	GEANY_KEYS_EDITOR_ADD_POSITION,				/**< Keybinding. */
```

Comment:
I don't like this name. Other navqueue keybindings are `GEANY_KEYS_GOTO_BACK` and `GEANY_KEYS_GOTO_FORWARD` -- nothing in `EDITOR`.  I'm not sure what a great name would be, maybe `GEANY_KEYS_NAVQUEUE_ADD_POSITION`, or maybe a shorter `GEANY_KEYS_NAVQUEUE_ADD_POS` or something.